File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

int cnt_P, cnt_Y;
bool solution(string s)
{
    for(int a = 0; a < s.size(); a++)
    {
        if(s[a] == 'p' || s[a] == 'P') ++cnt_P;
        else if(s[a] == 'y' || s[a] == 'Y') ++cnt_Y;
    }
    
    if(cnt_P == cnt_Y) return true;
    else               return false;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int pP = 0;
        int yY = 0;
        
        for(int i = 0; i < s.length(); i++) {
            if(s.charAt(i) == 'p' || s.charAt(i) == 'P') ++pP;
            else if (s.charAt(i) == 'y' || s.charAt(i) == 'Y') ++yY;
        }
        
        return pP == yY;
    }
}